modules/storage.js

```diff
@@ -0,0 +1,32 @@
+const Storage = {
+  DOMelements: {
+    email: document.getElementById('email'),
+    firstname: document.getElementById('firstName'),
+    lastname: document.getElementById('lastName'),
+    fullname: document.getElementById('fullname'),
+    message: document.getElementById('textArea'),
+  },
+  data: {
+    firstname: '',
+    lastname: '',
+    fullname: '',
+    email: '',
+    message: '',
+  },
+  set: (item, value) => {
+    Storage.data[item] = value;
+    window.localStorage.setItem('data', JSON.stringify(Storage.data));
+  },
+  init: () => {
+    if (window.localStorage.getItem('data')) {
+      const formInputs = JSON.parse(window.localStorage.getItem('data'));
+      Storage.data = formInputs;
+    }
+
+    Object.keys(Storage.DOMelements).forEach((element) => {
+      Storage.DOMelements[element].value = Storage.data[element];
+    });
+  },
+};
+
```

Comment:
* currently when  the form is filled error appears in the console please fix it 🙏 
![Screenshot (113)](https://user-images.githubusercontent.com/52497246/124183124-bb36e100-daaf-11eb-991d-e159407868f2.png)

Reply:
* This error still persists kindly fix it
* When the user loads the page or revisits  let data in the local storage be auto-populated with, the values entered in the previous visit to the page, currently this doesn't happen please fix